src/third_party/.gitignore

```diff
@@ -0,0 +1,2 @@
+libb2
```

Comment:
I would prefer to have only a top-level .gitignore, so please add the directories there instead.

Reply:
Absolutely, I'll make the changes.